custom/01-be-gone/cf_demo1.c

```diff
@@ -0,0 +1,99 @@
+#include "pf_all.h"      /* lots of stuff */
+#include "cf_helpers.h"  /* panic, safeAlloc, to_C_string, <stdio.h->{fprintf, stderr, ...}  */
+#include <errno.h>       /* errno         */
+
+/* 
+ * put forward declarations here if necessary
+*/
+
+
+/****************************************************************
+** Step 1: Put your own special glue routines here
+**     or link them in from another file or library.
+****************************************************************/
+
+/* exported functions */
+
+static cell_t f4711( cell_t Val )
+{/* a quick way to check that custom worlds are available
+ */
+    return 11 + 47*Val;
+}
+
+static cell_t be_gone( cell_t fileName, cell_t fnLen )
+{/* Demonstrates passing strings from PForth to C.
+    Interprets the passed strings as file name and tries to delete the file.
+	Returns 0 or errno 
+ */
+    int res;
+    char* buf = to_C_string( fileName, fnLen );
+    res = remove(buf);  /* delete file in file system */
```

Comment:
Deleting files is a bit dangerous for a demo.
Better to just do something innocuous.